tools/generator/src/Generator/SetTargetConfigurations.swift

```diff
@@ -357,7 +357,14 @@ $(CONFIGURATION_BUILD_DIR)
                 to: ldRunpathSearchPaths
             )
         }
-
+        
+        if buildMode == .xcode && target.platform.os == .iOS {
```

Comment:
@brentleyjones This feels a little gross, but assume limiting to iOS is the right thing here?

Am I also reading the fixtures right in that there isn't a current iOS resource fixture?

Reply:
There is a generated project that has it, which is enough coverage in my mind.

And yes, we should only set this for `buildMode == .xcode && target.product.type == .bundle && target.platform.os != .macOS`. You can extract that into an extension on `target`: `var shouldCodeSign: Bool` and then the check is `buildMode == .xcode && !target.shouldCodeSign`.

Reply:
https://github.com/buildbuddy-io/rules_xcodeproj/pull/1124#discussion_r975491405